src/main/scala/rps/Game.scala

```diff
@@ -1,57 +1,52 @@
 package rps
 
-object Move extends Enumeration {
-  type Move = Value
-  val Rock = Value("0")
-  val Paper = Value("1")
-  val Scissor = Value("2")
+import scala.util.Random
 
-  def isOrderType(s: String) = values.exists(_.toString == s)
+object Moves {
+  sealed class Move(val id: String, val name: String)
```

Comment:
suggestion: instead of defining a `Moves` object which contains a sealed `Move` class, you can define the subclasses in the [companion object](https://docs.scala-lang.org/tour/singleton-objects.html#companion-objects) of `Move`:

```scala
sealed class Move(val id: String, val name: String)
object Move {
  // ...
}
```

Also, I would use either a `sealed abstract class` or a `sealed trait` here, to avoid the possibility of instantiating `Move` directly.